lib/test/reporters.js

```diff
@@ -45,4 +46,13 @@ ObjectDefineProperties(module.exports, {
       return tap;
     },
   },
+  lcov: {
+    __proto__: null,
+    configurable: true,
+    enumerable: true,
+    get() {
+      lcov ??= require('internal/test_runner/reporter/lcov');
+      return lcov;
```

Comment:
We had [an issue](https://github.com/nodejs/node/issues/48112) opened when `spec` was exposed as a class which required instantiation.
Probably best to expose an instance instead of the class itself
```suggestion
      return ReflectConstruct(lcov, arguments);
```